new/LIT5843ShortPoemMary.xml

```diff
@@ -53,11 +52,19 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
     </teiHeader>
     <text>
         <body>
-            <div type="bibliography">
+             <div type="bibliography">
                 <listRelation>
                     <relation name="lawd:hasAttestation" active="LIT5843ShortPoemMary" passive="Neukirchen005"/>
                 </listRelation>
-            </div><!---->
+                <div type="textpart" corresp="Neukirchen005#ms_i3" xml:lang="gez">
+        <ab>
+          <l n="1">መኑ፡ ዘየኃድር፡ ውስተ፡ ለጽላሎትኪ፡ አርዝ፡</l>
```

Comment:
Do you want to indicate with `<l n="1">` that these are the first three verses of the poem, or on the page, or that there are linebreaks?

Reply:
These are three lines of the poem.